source/jobs/recover-jobs.txt

```diff
@@ -0,0 +1,49 @@
+.. _rm-sync-jobs-recoverability:
+
+==================
+Recover a Sync Job
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can recover continuous (CDC) sync jobs that 
+have failed within a 24-hour period. Procedures 
+for recovering a CDC sync job depend on if your using the embedded 
```

Comment:
(wording tweak)
The procedure for recovering a CDC sync job differs, depending on whether you're using ...